stable/pipelines/templates/argo/minio-secret.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.rbac.create }}
+apiVersion: v1
+data:
+  accesskey: bWluaW8=
+  secretkey: bWluaW8xMjM=
+kind: Secret
+metadata:
+  name: mlpipeline-minio-secret
```

Comment:
Even if not controllable via `_helpers`  and a dedicated value var, make ALL names have `.Release.Name` prefix please